includes/wikia/tasks/Tasks/MultiTask.class.php

```diff
@@ -214,6 +220,12 @@ private function getWikisWherePageExists( \Title $title ) {
 			$wikis[] = intval( $row->wiki_id );
 		}
 
+		if ( empty( $wikis ) ) {
+			$this->error( 'No wikis were found that have the provided page', [
```

Comment:
Is it an error?
It doesn't seem to be an error for me if I enter a non-existent page name however if we cannot fetch the list of wikis it is the error then. Can we somehow make sure these two cases are reported without any misleading?